hyperspy/tests/io/test_emd.py

```diff
@@ -67,6 +67,19 @@ def test_metadata():
     assert isinstance(signal, Signal2D)
 
 
+class TestMinimalSave():
+
+    def setup_method(self, method):
+        self.filename = 'testfile.emd'
```

Comment:
I would save the file in the tmp directory instead. In this way we may avoid future problems if we add more methods to this class and we run the tests in parallel. You could use e.g. `` with tempfile.TemporaryDirectory() as tmp:`` and this way there's no need for a tear down method.